sdk/go/common/workspace/plugins.go

```diff
@@ -1324,12 +1430,17 @@ func (d *pluginDownloader) downloadToFileWithRetry(pkgPlugin PluginSpec) (string
 	return path.(string), nil
 }
 
-// DownloadToFile downloads the given PluginSpec to a temporary file
-// and returns that temporary file.
-//
-// This has some retry logic to re-attempt the download if it errors for any reason.
-func (d *pluginDownloader) DownloadToFile(pkgPlugin PluginSpec) (*os.File, error) {
-	tarball, err := d.downloadToFileWithRetry(pkgPlugin)
+// DownloadToFile downloads the given PluginSpec to a temporary file and returns that temporary file. This has
+// some retry logic to re-attempt the download if it errors for any reason.
+func (d *pluginDownloader) DownloadToFile(ctx context.Context, pkgPlugin PluginSpec) (*os.File, error) {
```

Comment:
Context snuck in here
```suggestion
func (d *pluginDownloader) DownloadToFile(pkgPlugin PluginSpec) (*os.File, error) {
```